docs/api-reference/next/server.md

```diff
@@ -88,6 +88,22 @@ The introduction of the `307` status code means that the request method is prese
 
 The `redirect()` method uses a `307` by default, instead of a `302` temporary redirect, meaning your requests will _always_ be preserved as `POST` requests.
 
+### Accessing environment variables
+
+`process.env` holds an object with all environment variables used in Middleware, for both production and development environments. If you are not using any environment variables in your Middleware, the `process.env` object will be empty. `process.env` will only include environment variables _actually_ used.
```

Comment:
```suggestion
`process.env` can be used to access [environment variables](docs/basic-features/environment-variables.md) from Middleware. However, these are evaluated at build time so only environment variables _actually_ used will be included.
```

Its not clear what production and development means here (Vercel project settings or .env files??)

I think we can simplify this since it pertains to middleware only.